acceptance/main_test.go

```diff
@@ -0,0 +1,103 @@
+// Copyright 2024 Redpanda Data, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/go-logr/logr"
+	"github.com/redpanda-data/helm-charts/pkg/helm"
+	_ "github.com/redpanda-data/redpanda-operator/acceptance/steps"
+	framework "github.com/redpanda-data/redpanda-operator/harpoon"
+	redpandav1alpha1 "github.com/redpanda-data/redpanda-operator/src/go/k8s/api/redpanda/v1alpha1"
+	redpandav1alpha2 "github.com/redpanda-data/redpanda-operator/src/go/k8s/api/redpanda/v1alpha2"
+	"github.com/stretchr/testify/require"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+var (
+	imageRepo = "localhost/redpanda-operator"
+	imageTag  = "dev"
+	suite     *framework.Suite
+)
+
+func TestMain(m *testing.M) {
+	log.SetLogger(logr.Discard())
+
+	var err error
+
+	suite, err = framework.SuiteBuilderFromFlags().
+		RegisterProvider("eks", framework.NoopProvider).
+		RegisterProvider("gke", framework.NoopProvider).
+		RegisterProvider("aks", framework.NoopProvider).
+		RegisterProvider("k3d", framework.NoopProvider).
+		WithDefaultProvider("k3d").
+		WithSchemeFunctions(redpandav1alpha1.AddToScheme, redpandav1alpha2.AddToScheme).
+		WithHelmChart("https://charts.jetstack.io", "jetstack", "cert-manager", helm.InstallOptions{
```

Comment:
It would be better to run helm upgrade than install, so test execution would be idempotent